e2e/rbd.go

```diff
@@ -2072,6 +2079,35 @@ var _ = Describe("RBD", func() {
 
 			})
 
+			By("validate the functionality of controller with various encryptions", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
+				}
+				encryptionKMSIDList := []string{"", "secrets-metadata-test", "vault-test"}
+				for i := range encryptionKMSIDList {
```

Comment:
maybe use `for i, id := range encryptionKMSIDList` so that you can use `id` directly?